nvm.sh

```diff
@@ -781,7 +781,7 @@ nvm_strip_iojs_prefix() {
   if [ "${1-}" = "${NVM_IOJS_PREFIX}" ]; then
     nvm_echo
   else
-    nvm_echo "${1#"${NVM_IOJS_PREFIX}"-}"
+    nvm_echo "${1#${NVM_IOJS_PREFIX}-}"
```

Comment:
vim is incorrect - these nested quotes are quite valid.

You can test this on the command line with:
```sh
export A='iojs-1.2.3"
export NVM_IOJS_PREFIX='iojs'
echo "${A#"${NVM_IOJS_PREFIX}"-}"
```
and it will output "1.2.3"